cache/memory_test.go

```diff
@@ -67,23 +67,23 @@ func (s *MemoryTestSuite) TestDecrement() {
 }
 
 func (s *MemoryTestSuite) TestDecrementWithConcurrent() {
-	res, err := s.memory.Decrement("decrement")
+	res, err := s.memory.Decrement("decrement_concurrent")
```

Comment:
Is there any difference between `decrement` and `decrement_concurrent`?

Reply:
> Is there any difference between `decrement` and `decrement_concurrent`?

No more, but in the redis test case, the data is not cleared, so the test results are inconsistent.